common/app/views/fragments/amp/stylesheets/buttons.scala.html

```diff
@@ -30,10 +30,18 @@
     margin: 0 20px 0;
 }
 
+.cta--show-more.cta--show-more__unindent {
```

Comment:
I am not sure we want to unindent this. When we add padding/margin around the outbrain container to make it inline with the rest of the components, it shouldn't include the overflow element.